keyboards/heliar/wm1_hotswap/wm1_hotswap.c

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2019 HELIAR MK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "wm1_hotswap.h"
+
+
+void keyboard_pre_init_kb(void)
+{
+    setPinOutput(D7);
+    writePinHigh(D7);
+    setPinOutput(D6);
+    writePinHigh(D6);
+    setPinOutput(D4);
+    writePinHigh(D4);
```

Comment:
IIRC, the `led_update_kb` function is called on startup, so, you don't need to set the default state.  You can just set the pins.

Reply:
It's not, it first gets called some short time after USB enumeration. This is more than enough time to be visible (doing this fixed it on the Wasdat, though it's in `matrix_init_kb()` instead).

Reply:
So, the conclusion is we should set the default state at this function, right?

Reply:
Yes, it should be fine as is. You can test without the `writePinHigh()`s if you like, to see whether the LEDs flicker when you plug the board in.

Reply:
Ok. That should be fine. So there is no need to change this code block.